### ISSUE TITLE
handling more than one image belonging to one patient

### ISSUE THREAD

**Issue by dilekozkan** [MIC-DKFZ/nnUNet#2660]
Hi!
The point I don't understand is that how UNet supports png.
I have more than one png file belonging to a patient. should I leave them separately, or collect all the files of a patient into nii.gz?
which one true? (one modality OCT - duke dataset)

**Comment by dilekozkan** [comment]
@FabianIsensee  (thanks in advance, happy new years!!)
**for the details:**
I am using Duke OCT dataset. we have one modality. Actually 61 scans are obtained for each patient. But since there is no significant difference in consecutive images, doctors created a label for every 5 slices. 11 slices are annotated for 10 patients. (so 110 images and 110 labels are shared)
We have separate png slice files for each patient.
Should I convert this to a nii.gz format (**not exactly a consecutive CT though?**)
or should I let nnUnet treat each slice of each patient as a separate sample

Duke Dataset Reference:
P. P. Srinivasan, L.A. Kim, P.S. Mettu, S.W. Cousins, G.M. Comer, J.A. Izatt, and S. Farsiu,
"Fully automated detection of diabetic macular edema and dry age-related macular degeneration from optical coherence tomography images",BioMedical Optics Express,5(10), pp. 3568-3577, 2014

**Comment by mrokuss** [comment]
Hey @dilekozkan 

In your case it probably makes the most sense to treat each slice separately even if it is from the same patient. However, one important thing to keep in mind is that you should then use custom splits where the images from one patient are only ever present in one respective split. You can use [GroupedKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupKFold.html) for that.

Hope this helps!
